### PR TITLE
Fix empty error messages

### LINKED ISSUE #15782: Review and change game action result messages
A lot of game actions use no messages or some sort of substitute, all of them should be reviewed after #15637 is merged.

An example:
https://github.com/OpenRCT2/OpenRCT2/blob/a4ed1cb6ced5e036e0a29f5d562a3326f31cd796/src/openrct2/actions/BalloonPressAction.cpp#L51-L55

Right now it has no title or error message, this should ideally say something like "Invalid entity index" which could be STR_INVALID_PARAMETER for the title and STR_INVALID_ENTITY_INDEX for the message.

This is just an example, ideally one extracts all lines with inappropriate messages to make a nice table.

### PR DESCRIPTION
**Pull request by pfroud**
Closes #15782

### CODE REVIEW

File: src/openrct2/actions/ParkSetEntranceFeeAction.cpp [hunk 1]
```diff
@@ -46,7 +46,8 @@ GameActions::Result ParkSetEntranceFeeAction::Query() const
     bool forceFreeEntry = !ParkEntranceFeeUnlocked();
     if (noMoney || forceFreeEntry)
     {
-        return GameActions::Result(GameActions::Status::Disallowed, STR_NONE, STR_NONE);
+        return GameActions::Result(
+            GameActions::Status::Disallowed, STR_ERR_INVALID_PARAMETER, STR_ERR_CANT_CHANGE_PARK_ENTRANCE_FEE);
```
[the remark this answers is not in the document]
Comment (pfroud) [reply src/openrct2/actions/ParkSetEntranceFeeAction.cpp]: Good point, I changed it to

```C++
return GameActions::Result(GameActions::Status::Disallowed, STR_ERR_CANT_CHANGE_PARK_ENTRANCE_FEE, STR_NONE);
```

**Review by Gymnasiast — APPROVED** [review]

**Review by tupaschoal — COMMENTED** [review]
@pfroud can you rebase your branch to solve merge conflicts on the strings file please?

**Comment by pfroud** [comment]
OK, it will be my first time using git rebase.

I read the instructions here: https://github.com/OpenRCT2/OpenRCT2/wiki/Rebase-and-Sync-fork-with-OpenRCT2#solve-merge-conflicts

To be clear, when rebase is running, I should change `pick` to `edit` to change the stringIDs in my fork, right?

**Comment by Gymnasiast** [comment]
Yes.

(Generally, changed string IDs will cause merge conflicts anyway, so pick usually works here too, but `edit` is more predictable.)

**Comment by tupaschoal** [comment]
Were you able to do it @pfroud? Do you need help?

**Comment by pfroud** [comment]
I just did it, it was easy after all! Thanks for checking in :)

**Review by Gymnasiast — APPROVED** [review]

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
I merged and just saw that it had the changelog label. Not sure if it got lost in the rebase or if we ever added one. Do you think it's needed @Gymnasiast ? If it is, can you make another PR fixing that @pfroud ?

**Comment by pfroud** [comment]
There never was a changelog entry, I will add one if we decide it is needed

**Comment by Gymnasiast** [comment]
Yes please.

**Comment by Gymnasiast** [comment]
No, there should be an entry for the release it was introduced. That can be done retro-actively.

**Comment by Gymnasiast** [comment]
Created https://github.com/OpenRCT2/OpenRCT2/pull/21725 to add the changelog entry.

**Pull request merged**